src/app/workflow/snapshot-exporter-modal/snaphot-exporter-modal.component.scss

```diff
@@ -0,0 +1,37 @@
+@import 'materialColorScheme.scss';
+
+.confirmation-box {
```

Comment:
Since this style is component specific, need an expanation on why it has (and plans to have) a different confirmation style than everything else on the site.